### ISSUE TITLE
Disabling immersive mode from haxe side?

### ISSUE THREAD

**Issue by rottor** [fbricker/extension-android-immersive#2]
Is there any way to disable immersive mode from haxe side?
For example, if I want system UI to be shown again if player press some button.

**Comment by joackobengochea** [comment]
Do you mean like the extension wasn't included? the extension allows you to
show the ui when the user triggers it (for example, sliding down the top of
the screen) and hides it automatically a few seconds after if there's no
more interactions with it.

2015-12-04 14:23 GMT-03:00 Yuri K notifications@github.com:

**Comment by rottor** [comment]
Not exactly. I need to use custom trigger from my UI, not the default sliding down the top of
the screen and so on.

**Comment by joackobengochea** [comment]
i don't know if that's possible since that events are handled by the
android os... anyway, you can look at the code of this library to see how
the events are handled, and try to implement a library for handling custom
triggers, or send a pull request to this library adding that functionality

2015-12-08 8:46 GMT-03:00 Yuri K notifications@github.com:
